docs/components/gantry/single-axis.md

```diff
@@ -59,11 +59,82 @@ The following attributes are available for `single-axis` gantries:
-| `limit_pins`  | Optional | object | The `boards`'s pins attached to the limit switches on either end. If the [motor](/components/motor/) used does not include an [encoder](/components/motor/gpio/encoded-motor/), `limit_pins` are required to be set. |
-| `limit_pin_enabled_high` | boolean | Optional | Whether the limit pins are enabled. <br> Default: `false` |
+| `board`  |  string | Optional | The `name` of the [board](/components/board/) containing the limit switches and pins. If `limit_pins` is set, `board` is **required**. |
+| `limit_pins`  | object | Optional | The pins on the `board` which are attached to the limit switches on either end. If the [motor](/components/motor/) used does not include an [encoder](/components/motor/gpio/encoded-motor/), you must set `limit_pins`. The order of these pins is important. If the pins are in the wrong order, the gantry may try to travel beyond the limit switch (or switches). The switch representing the home (`0`) position must be first in the list. In addition, if you are configuring multiple `single-axis` gantries, you can [specify the order of homing routines](#specify-the-order-of-homing-routines) if needed. |
+| `limit_pin_enabled_high` | boolean | Optional | Whether the limit pin must be “high” or “low” to be considered “hit”, with a value of `true` representing "high". This attribute is **required** if `limit_pins` is set.<br> Default: `false` |
 | `mm_per_rev` | int | Optional | How far the gantry moves (linear, distance in mm) per one revolution of the motor’s output shaft. This typically corresponds to Distance = PulleyDiameter * pi, or the pitch of a linear screw. |
 | `gantry_rpm` | int | Optional | The gantry `motor`'s default revolutions per minute (RPM). |
+
+## Specify the order of homing routines
+
+When `limit_pins` is set, a `single-axis` gantry component performs a homing routine when `viam-server` starts.
+The axis will move towards one limit switch, then the other, before stopping at 80% of the travel of the axis.
+
+If there are multiple `single-axis` gantries configured (for example, when using a [`multi-axis` gantry](/components/gantry/multi-axis/)), the order in which these homing routines are executed may be important.
+
+In this case, you can specify the axes that must be homed first in a `depends_on` array to each `single-axis` gantry.
+For example, if there is a tool mounted to the z axis that could be damaged during homing of the x or y axes, you should add the z axis as a dependency to the x and y axes in your configuration. This will ensure the z axis completes its homing routine first, and that the tool head returns to a safe location to begin the homing process on the other axes.
+
+{{% alert="Note" color="note" %}}
+The order of axes in the `subaxes_list` for a `multi-axis` gantry does not influence the order of homing.
+A `multi-axis` gantry component is not constructed until all the `single-axis` gantry components are constructed and finish homing.
```

Comment:
```suggestion
A `multi-axis` gantry component is not constructed until all the `single-axis` gantry components are constructed and finish homing.
However, the order of axes in the `subaxes_list` for a `multi-axis` gantry does not influence the order of homing.
```
I think this makes a little more sense? I think the existing second sentence is a slightly separate concept from the existing first and last sentences, so maybe it can actually come out of this note box entirely? Or consider reordering thusly.

Reply:
I see what you're saying! However, I think removing the second sentence entirely is the play here: it's an implementation detail that the user won't really interact with (how we construct mult-axis components on our end). From the user perspective: they just need to know that homing order is controllable only with `depends_on`, and not by any ordering of the `subaxes_list` to `multi-axis` (which, conceptually, could otherwise seem a logical assumption). Pulling that out seems to make this admonition much more focused and clearer -- LMK if you agree!

Reply:
Yes, I do agree! That part looks good